.github/workflows/ci.yml

```diff
@@ -144,5 +144,14 @@ jobs:
           echo "$PGP_PASSPHRASE" | gpg --pinentry-mode loopback --passphrase-fd 0 --import /tmp/signing-key.gpg
           (echo "$PGP_PASSPHRASE"; echo; echo) | gpg --command-fd 0 --pinentry-mode loopback --change-passphrase $(gpg --list-secret-keys --with-colons 2> /dev/null | grep '^sec:' | cut --delimiter ':' --fields 5 | tail -n 1)
 
+      - uses: actions/setup-node@v3
```

Comment:
Will this run on tag?

Reply:
It's part of the usual maven publish job, so it will run whenever that runs.